csun-metalab/nomi
# NAM-545 -- UX: Application must be able to "tab" through

# Description
Added ability to tab through most of the "intractables".
  
# Testing
1. Visit each route; click the top nav-bar(resets the tab position) and tab through the page.
2. Ensure tabbing through our application is somewhat passable.
